app/Http/Controllers/Staff/Contacts/Email/UpdateAction.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers\Staff\Contacts\Email;
+
+use App\Eloquents\ContactEmail;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Staff\Contacts\Email\EmailRequest;
+use App\Services\Contacts\ContactsService;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Mail;
+
+class UpdateAction extends Controller
+{
+
+    /**
+     * @var contactsService
```

Comment:
ここに書くのはクラス名です！大文字小文字の区別もお願いします！

ちなみに、このコメントを書いている理由なんですが、VS Code の [PHP IntelliSense](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-intellisense) という拡張機能を入れると、メソッド名の補完が効くからです

```suggestion
     * @var ContactsService
```